top.q

```diff
@@ -0,0 +1,13 @@
+if[2=count .z.x;(`$"::",.z.x 0).z.i;system"l ",.z.x 1;exit 0]
```

Comment:
Can you add a reference line to the top of this i.e. where it comes from in code.kx.com
Can you also add something which makes this exit with a verbose error message if somebodies tries to run this with .z.K < 4

Reply:
I assume this also only works on Linux, maybe add an error message as well if someone tries to run it on Windows? (not sure if it works on Mac). I know we're adding it to `torq.sh` but maybe people will see `top.q` & try to run it manually on Windows.